mbed_lstools/main.py

```diff
@@ -130,6 +130,9 @@ def mock_platform(mbeds, args):
         else:
             logger.error("Could not parse mock from token: '%s'", token)
 
+def update_from_web(mbeds, args):
+    mbeds.plat_db.update_from_web()
```

Comment:
would be good to indicate if update success or not as a retcode

Reply:
Agreed